spec/index.bs

```diff
@@ -2465,10 +2459,7 @@ Issue(https://github.com/gpuweb/gpuweb/issues/851): consider making `textureComp
 
     : <dfn>viewDimension</dfn>
     ::
-        If the {{GPUBindGroupLayoutEntry/type}} is
-        {{GPUBindingType/"sampled-texture"}},
-        {{GPUBindingType/"readonly-storage-texture"}}, or
-        {{GPUBindingType/"writeonly-storage-texture"}}:
```

Comment:
I assume that this list was previously supposed to be updated to include `"multisampled-texture"`, yes?

Reply:
correct!